src/main.rs

```diff
@@ -142,8 +144,15 @@ fn init_tracing() {
         }
     }
 
-    tracing_subscriber::fmt::Subscriber::builder()
-        .with_env_filter(env_filter)
-        .with_writer(std::io::stderr)
-        .init();
+   let (non_blocking_writer, _guard) = tracing_appender::non_blocking(std::io::stderr());
+   
+   let subscriber = tracing_subscriber::FmtSubscriber::builder()
+   .with_env_filter(env_filter)
+   .with_writer(non_blocking_writer)
+   .finish();
+
+   DefaultSubcriberGuard {
```

Comment:
To see why we return the subscriber and writer_guard, read this quick paragraph [here](https://docs.rs/tracing-appender/0.1.1/tracing_appender/non_blocking/struct.WorkerGuard.html).

Reply:
Thanks for the explanation!
Let's have it in the comments in the code? In 3 months we all forget about it